app/page-partials/my-dashboard/UserRoles.tsx

```diff
@@ -281,6 +281,8 @@ const UserRoles = ({ selectedRequest, alert }: Props) => {
     );
   }
 
+  const showIdirLookupOption = selectedIdp === 'idir' && (idirSearched || selectedProperty !== 'guid');
```

Comment:
Could you add some parentheses here to make the logic a  bit more clear please?

Reply:
hm.. I do not think this level of conditional logic is difficult to understand and having the equivalent logic in 3-4 lines helps.
do you have proposed code changes though?

Reply:
`showIdirLookupOption = (selectedIdp === 'idir') && (idirSearched || selectedProperty !== 'guid');` 
Is more clear to me.  I was able to sort out the logic, but all the and and || operators made it difficult to parse on a first read for someone who hadn't wrote it